### PR TITLE
Move chat app to run with default scripts

### PR DESCRIPTION
**Pull request by misslivirose**
Fogbugz case 5153

Greeters and users in-world have asked for text chat to be a default application so that new users (especially those with audio challenges) are better able to troubleshoot quickly. 

So people can choose to remove the chat app if they don't want it, I've added it to the list of separate default scripts. 

Test plan:
Launch a clean version of Interface. Ensure that chat is part of the default scripts and runs as such. Observe chat works as intended:
- Tablet UI
- HUD UI
- After reloading scripts

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/503/HighFidelity-Beta-PR10647-03edb06ecb9868da944156a5fb486312e171f135.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/503/HighFidelity-Beta-PR10647-03edb06ecb9868da944156a5fb486312e171f135.dmg

File: scripts/system/chat.js [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 (function() {
 
-    var webPageURL = "ChatPage.html"; // URL of tablet web page.
+    var webPageURL = Script.resolvePath("html/ChatPage.html");; // URL of tablet web page.
```
Comment (sethalves) [inline scripts/system/chat.js]: stray semicolon here?

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
Testing complete: 

Test plan:
Launch a clean version of Interface. Ensure that chat is part of the default scripts and runs as such. Observe chat works as intended:

Tablet UI **Pass**
HUD UI **Pass**
After reloading scripts **Pass**

Please let me know if you require any further assistance :)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/526/HighFidelity-Beta-PR10647-6657c4d997302e100edf15262697630eaff7a28f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/526/HighFidelity-Beta-PR10647-6657c4d997302e100edf15262697630eaff7a28f.dmg

**Comment by tapalisa** [comment]
This is a big product decision and should not have been implemented and merged without discussion from the product team.

**Comment by sethalves** [comment]
Okay.  It's easily undone.  Let us know what should happen.

**Pull request merged**